closure/goog/math/integer.js

```diff
@@ -589,6 +643,13 @@ goog.math.Integer.prototype.divide = function(other) {
     return this.divide(other.negate()).negate();
   }
 
+  // Have to degrade to slowDivide for Very Large Numbers, because
+  // they're out of range for the floating-point approximation
+  // technique used below.
+  if(this.bits_.length > 30) {
```

Comment:
Sorry about the style issues; I couldn't get the closure-linter to install. Hopefully there aren't more issues.